doc/Protocols/SecurityServerExtension/pr-targetss_security_server_targeting_extension_for_the_x-road_protocol.md

```diff
@@ -0,0 +1,241 @@
+
+The XML Schema for this extension is listed in the section [XML Schema for the extension](#xml-schema-for-the-extension).
+
+
+### Schema header
+
+The following listing shows the header of the schema definition
+
+```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema elementFormDefault="qualified"
+        targetNamespace="http://x-road.eu/xsd/xroad.xsd"
+        xmlns="http://x-road.eu/xsd/xroad.xsd"
+        xmlns:id="http://x-road.eu/xsd/identifiers"
+        xmlns:xs="http://www.w3.org/2001/XMLSchema">
+    <xs:import namespace="http://www.w3.org/XML/1998/namespace"
+            schemaLocation="http://www.w3.org/2001/xml.xsd"/>
+    <xs:import id="id" namespace="http://x-road.eu/xsd/identifiers"
+            schemaLocation="http://x-road.eu/xsd/identifiers.xsd"/>
+</xs:schema>
+
+```
+
+### Added `securityServer` element
+A new `securityServer` element was added to identify the specific target security server.
+
+```xml
+ <xs:element name="securityServer" type="id:XRoadSecurityServerIdentifierType">
+        <xs:annotation>
+            <xs:documentation>Identifies a specific security server</xs:documentation>
+        </xs:annotation>
+    </xs:element>
+```
+  The element is of the type `XRoadSecurityServerIdentifierType`, which is one of the identifiers already defined
+  in the X-Road Message Protocol v 4.0 \[[PR-MESS](#Ref_PR-MESS)\] section 2.1. The whole XML schema for the identifier types is defined in
+  Annex A of the same document. The relevant part is listed below for convenience.
+
+```xml
+<xs:complexType name="XRoadSecurityServerIdentifierType">
+    <xs:complexContent>
+        <xs:restriction base="XRoadIdentifierType">
+            <x:sequence>
+                <xs:element ref="xRoadInstance"/>
+                <xs:element ref="memberClass"/>
+                <xs:element ref="memberCode"/>
+                <xs:element ref="serverCode"/>
+            </xs:sequence>
+            <xs:attribute ref="objectType" use="required" fixed="SERVER"/>
+        </xs:restriction>
+    </xs:complexContent>
+</xs:complexType>
+```
+
+### Message headers
+ This section describes the additional SOAP headers that are added by this extension.
+
+|Field | Type | Mandatory/Optional | Description |
+|-------------|-------------|-------------|-------------|
+| securityServer | XRoadSecurityServerIdentifierType | Optional | The security server this message is for |
+
+
+## XML Schema for the extension
+ ```xml
+<?xml version="1.0" encoding="UTF-8"?>
+<xs:schema elementFormDefault="qualified"
+        targetNamespace="http://x-road.eu/xsd/xroad.xsd"
+        xmlns="http://x-road.eu/xsd/xroad.xsd"
+        xmlns:id="http://x-road.eu/xsd/identifiers"
+        xmlns:xs="http://www.w3.org/2001/XMLSchema">
+    <xs:import namespace="http://www.w3.org/XML/1998/namespace"
+            schemaLocation="http://www.w3.org/2001/xml.xsd"/>
```

Comment:
http://x-road.eu/xsd/xroad.xsd is using http://www.w3.org/2009/01/xml.xsd schema location, and to avoid validation problems extension must use the same.